### ISSUE TITLE
Standalone persistance?

### ISSUE THREAD

**Issue by zevv** [geoffreybennett/alsa-scarlett-gui#54]
Wow, nice to see proper linux support for my brand new Scarlett - having this little gui is easier than trying to tame the pipewire session manager.

4i4, properly detected, I'm able to alter the routing.

```
Bus 003 Device 015: ID 1235:8212 Focusrite-Novation Scarlett 4i4 USB
```

I'm enabling "Standalone" mode in the startup menu, but when the device loses USB power and is reconnected, the routing does not persist and falls back to the default.

I assume I'm doing something wrong here, but not sure what?

**Comment by lkamp** [comment]
Kind of guessing here, but I think the settings are made persistent on your device when you shut down your computer (more correctly when ALSA stops, but rebooting is the fastest way for me). This is to prevent the flash memory on your Scarlett from wearing out when you change settings a lot.

So, can you try to set your desired routing, then reboot with your Scarlett connected to the PC and see if that changes anything?

**Comment by zevv** [comment]
Right, I found this is indeed the problem. My "fix" is to simply restore the alsa mixer to my preferred settings with `amix` after a suspend/resume; this just another task for the hacky script that I already had to turn down the fans and rainbow blinkenlights on my motherboard, restore my mouse DPI, disable the caps lock on my keyboard and tweak the brightness of my display: all kinds of tasks that I probably am supposed to configure somewhere deep in the caverns of modern linux desktop environments.

Thanks!
